File: backend/java/spring-boot/demo/src/main/java/com/example/demo/ctrl/DemoCtrl.java
package com.example.demo.ctrl;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class DemoCtrl {
  
  @GetMapping("/")
  public String helloWorld() {
    return "{\"msg\":\"Hello World\"}";
  }
}

File: kotlin/spring-boot/demo/src/main/kotlin/com/example/demo/ctrl/DemoCtrl.kt
package com.example.demo.ctrl

import org.springframework.web.bind.annotation.RestController
import org.springframework.web.bind.annotation.GetMapping

@RestController
class DemoCtrl {
  
  @GetMapping("/")
  fun helloWorld(): String {
    return "{\"msg\":\"Hello World\"}"
  }
}

